File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/dto/SysUserInfoDTO.java
package ink.ckx.mo.system.api.model.dto;

import ink.ckx.mo.common.web.enums.StatusEnum;
import lombok.Data;

import java.util.Set;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/11
 */
@Data
public class SysUserInfoDTO {

    /**
     * 用户id
     */
    private Long userId;

    /**
     * 部门id
     */
    private Long deptId;

    /**
     * 用户id
     */
    private Integer dataScope;

    /**
     * 用户名
     */
    private String username;

    /**
     * 用户密码
     */
    private String password;

    /**
     * 用户状态
     */
    private StatusEnum status;

    /**
     * 用户角色编码集合
     */
    private Set<String> roles;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/dto/SysUserInfoDTO.kt
package ink.ckx.mo.admin.api.model.dto

import ink.ckx.mo.common.mybatis.enums.DataScopeEnum
import ink.ckx.mo.common.web.enums.StatusEnum

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/11
 */
data class SysUserInfoDTO(

    /**
     * 用户id
     */
    var userId: Long? = null,

    /**
     * 部门id
     */
    var deptId: Long? = null,

    /**
     * 数据权限
     */
    var dataScope: DataScopeEnum? = null,

    /**
     * 用户名
     */
    var username: String? = null,

    /**
     * 用户密码
     */
    var password: String? = null,

    /**
     * 用户状态
     */
    var status: StatusEnum? = null,

    /**
     * 用户角色编码集合
     */
    var roles: Set<String>? = null,
)